src/GraphQl/Resolver/Stage/SerializeStage.php

```diff
@@ -117,53 +116,61 @@ private function serializeCursorBasedPaginatedCollection(iterable $collection, a
             throw new \LogicException(sprintf('Collection returned by the collection data provider must implement %s or %s.', PaginatorInterface::class, PartialPaginatorInterface::class));
         }
 
+        $selection = $context['info']->getFieldSelection(1);
```

Comment:
are we sure `info` always exists?

Reply:
For calls in api-p, in the context of a collection, yes. The [ReadStage](https://github.com/api-platform/core/blob/main/src/GraphQl/Resolver/Stage/ReadStage.php#L78) uses it without any test. Should we enforce it here ?